src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const cssProperties = sourceString
+    .trim()
+    .split(';')
+    .filter((str) => str.trim())
+    .map((str) => {
+      return str
+        .trim()
+        .split(':')
+        .map((s) => s.trim());
+    });
 
+  return Object.fromEntries(cssProperties);
```

Comment:
The code is clean and follows the checklist well. However, consider handling edge cases where there might be multiple semicolons without any styles between them, as this could lead to empty entries in the resulting object.